src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -189,11 +190,14 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
               executionInsert.executionId.get,
               None,
               None)
+        case j: JesBackend =>
+          dataAccess.localJobsAutoInc += new LocalJob(executionInsert.executionId.get, None, None) // FIXME: Placeholder?
```

Comment:
Doesn't appear there are any current `DataAccess` callers that are actually _using_ these tables, but this should eventually be `dataAccess.jesJobsAutoInc`, not `localJobsAutoInc`.